### DISCUSSION TITLE
Drop all itens in inventory

### DISCUSSION THREAD

**Discussion by ruizaoo** [PrismarineJS/mineflayer#2855] [Q&A]
Can anybody help me?

I'm trying to make bots drop all inventory items using a discord command, but the bot only drops one item and crashes

this is the code.

---

   else if (message.content.startsWith('+drop')) {
      let split = message.content.split(' ')
      if (split[1] == bot.username || split[1] == 'all') {
        channel.send(`Dropping Items: ${username}`)
        for (item of bot.inventory.items()) {
            bot.tossStack(item);
            await utils.sleep(3000);
        }
        return;
      }
    }

**Comment by amoraschi** [comment]
Fill out the issue form
